app/templates/components/category-list.hbs

```diff
@@ -0,0 +1,7 @@
+{{yield}}
+<ul class="collection">
```

Comment:
I thought about that, but was stopped by naming conventions: 
"Assertion Failed: You cannot use 'categories' as a component name. Component names must contain a hyphen."